### ISSUE TITLE
IncompatibleClassChangeError when invoking gradle revapi

### ISSUE THREAD

**Issue by ghilainm** [palantir/gradle-revapi#147]
I apply the plugin and invoked gradle revapi and the following error occured:

```
java.lang.IncompatibleClassChangeError
        at com.palantir.gradle.revapi.GradleUtils$$Lambda$1728.0000000064801A80.call(Unknown Source)
```

I am using the following version:
    id "com.palantir.revapi" version "1.1.3"

I have also tried with version 1.1.2

**Comment by CRogers** [comment]
Thanks for the report!

Which version of Gradle are you using? Do you have a repo available I can look at or a minimal repro that I can see?

From looking at the code, I suspect it's some sort of Guava classpath conflict.

**Comment by iamdanfox** [comment]
To diagnose classpath conflicts between _plugins_, you can run `./gradlew buildEnvironment`.

**Comment by ghilainm** [comment]
@CRogers I though also about an issue with guava.

I have no opensource repository that you can use to reproduce the issue sorry. Here is the output of the command you want to see.

```
+--- luccs-openapi:luccs-openapi.gradle.plugin:2.0.2
|    \--- lu.etat.ada.luccs.framework.plugins.gradle:fwk-luccs-openapi-plugin:2.0.2
|         +--- lu.etat.ada.luccs.framework.plugins.gradle:fwk-luccs-library-plugin:2.0.2
|         |    +--- lu.etat.ada.luccs.framework.plugins.gradle:fwk-luccs-common-plugin:2.0.2
|         |    |    +--- gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.2.0
|         |    |    |    \--- org.ajoberstar.grgit:grgit-core:3.1.1
|         |    |    |         \--- org.eclipse.jgit:org.eclipse.jgit:5.3.0.201903130848-r
|         |    |    |              +--- com.jcraft:jsch:0.1.54
|         |    |    |              +--- com.jcraft:jzlib:1.1.1
|         |    |    |              +--- com.googlecode.javaewah:JavaEWAH:1.1.6
|         |    |    |              +--- org.slf4j:slf4j-api:1.7.2 -> 1.7.28
|         |    |    |              +--- org.bouncycastle:bcpg-jdk15on:1.60
|         |    |    |              |    \--- org.bouncycastle:bcprov-jdk15on:1.60
|         |    |    |              +--- org.bouncycastle:bcprov-jdk15on:1.60
|         |    |    |              \--- org.bouncycastle:bcpkix-jdk15on:1.60
|         |    |    |                   \--- org.bouncycastle:bcprov-jdk15on:1.60
|         |    |    +--- pl.allegro.tech.build:axion-release-plugin:1.10.2
|         |    |    |    +--- org.eclipse.jgit:org.eclipse.jgit:4.11.0.201803080745-r -> 5.3.0.201903130848-r (*)
|         |    |    |    +--- org.eclipse.jgit:org.eclipse.jgit.ui:4.11.0.201803080745-r
|         |    |    |    |    \--- org.eclipse.jgit:org.eclipse.jgit:4.11.0.201803080745-r -> 5.3.0.201903130848-r (*)
|         |    |    |    +--- com.jcraft:jsch:0.1.54
|         |    |    |    +--- com.jcraft:jsch.agentproxy.core:0.0.9
|         |    |    |    +--- com.jcraft:jsch.agentproxy.jsch:0.0.9
|         |    |    |    |    +--- com.jcraft:jsch:0.1.49 -> 0.1.54
|         |    |    |    |    \--- com.jcraft:jsch.agentproxy.core:0.0.9
|         |    |    |    +--- com.jcraft:jsch.agentproxy.sshagent:0.0.9
|         |    |    |    |    \--- com.jcraft:jsch.agentproxy.core:0.0.9
|         |    |    |    +--- com.jcraft:jsch.agentproxy.pageant:0.0.9
|         |    |    |    |    +--- com.jcraft:jsch.agentproxy.core:0.0.9
|         |    |    |    |    +--- net.java.dev.jna:jna:4.1.0
|         |    |    |    |    \--- net.java.dev.jna:jna-platform:4.1.0
|         |    |    |    |         \--- net.java.dev.jna:jna:4.1.0
|         |    |    |    +--- com.jcraft:jsch.agentproxy.usocket-jna:0.0.9
|         |    |    |    |    +--- com.jcraft:jsch.agentproxy.core:0.0.9
|         |    |    |    |    +--- net.java.dev.jna:jna:4.1.0
|         |    |    |    |    \--- net.java.dev.jna:jna-platform:4.1.0 (*)
|         |    |    |    +--- com.jcraft:jsch.agentproxy.usocket-nc:0.0.9
|         |    |    |    |    \--- com.jcraft:jsch.agentproxy.core:0.0.9
|         |    |    |    \--- com.github.zafarkhaja:java-semver:0.9.0
|         |    |    \--- io.github.http-builder-ng:http-builder-ng-core:1.0.4
|         |    |         +--- xml-resolver:xml-resolver:1.2
|         |    |         \--- org.slf4j:slf4j-api:1.7.21 -> 1.7.28
|         |    +--- lu.etat.ada.luccs.framework.plugins.gradle:fwk-luccs-test-plugin:2.0.2
|         |    \--- com.github.ben-manes:gradle-versions-plugin:0.22.0
|         |         \--- com.thoughtworks.xstream:xstream:1.4.10
|         |              +--- xmlpull:xmlpull:1.1.3.1
|         |              \--- xpp3:xpp3_min:1.1.4c
|         +--- org.openapitools:openapi-generator-gradle-plugin:4.2.1
|         |    +--- org.openapitools:openapi-generator:4.2.1
|         |    |    +--- io.swagger.core.v3:swagger-core:2.0.10
|         |    |    |    +--- javax.xml.bind:jaxb-api:2.3.0
|         |    |    |    +--- org.apache.commons:commons-lang3:3.7
|         |    |    |    +--- org.slf4j:slf4j-api:1.7.25 -> 1.7.28
|         |    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.10 -> 2.10.2
|         |    |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.9.10 -> 2.10.2
|         |    |    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.10.2
|         |    |    |    |    \--- com.fasterxml.jackson.core:jackson-core:2.10.2
|         |    |    |    +--- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.10 -> 2.10.2
|         |    |    |    |    +--- org.yaml:snakeyaml:1.24
|         |    |    |    |    \--- com.fasterxml.jackson.core:jackson-core:2.10.2
|         |    |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.9.10
|         |    |    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.10 -> 2.10.2
|         |    |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.10 -> 2.10.2
|         |    |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.9.10 -> 2.10.2 (*)
|         |    |    |    +--- io.swagger.core.v3:swagger-annotations:2.0.10
|         |    |    |    +--- io.swagger.core.v3:swagger-models:2.0.10
|         |    |    |    |    \--- com.fasterxml.jackson.core:jackson-annotations:2.9.10 -> 2.10.2
|         |    |    |    \--- javax.validation:validation-api:1.1.0.Final
|         |    |    +--- io.swagger.parser.v3:swagger-parser:2.0.15
|         |    |    |    +--- io.swagger.parser.v3:swagger-parser-v2-converter:2.0.15
|         |    |    |    |    +--- io.swagger:swagger-parser:1.0.47
|         |    |    |    |    |    +--- io.swagger:swagger-core:1.5.24
|         |    |    |    |    |    |    +--- org.apache.commons:commons-lang3:3.2.1 -> 3.7
|         |    |    |    |    |    |    +--- org.slf4j:slf4j-api:1.7.22 -> 1.7.28
|         |    |    |    |    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.10 -> 2.10.2
|         |    |    |    |    |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.9.10 -> 2.10.2 (*)
|         |    |    |    |    |    |    +--- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.10 -> 2.10.2 (*)
|         |    |    |    |    |    |    +--- io.swagger:swagger-models:1.5.24
|         |    |    |    |    |    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.10 -> 2.10.2
|         |    |    |    |    |    |    |    +--- org.slf4j:slf4j-api:1.7.22 -> 1.7.28
|         |    |    |    |    |    |    |    \--- io.swagger:swagger-annotations:1.5.24
|         |    |    |    |    |    |    +--- com.google.guava:guava:27.0.1-android
|         |    |    |    |    |    |    |    +--- com.google.guava:failureaccess:1.0.1
|         |    |    |    |    |    |    |    +--- com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava
|         |    |    |    |    |    |    |    +--- com.google.code.findbugs:jsr305:3.0.2
|         |    |    |    |    |    |    |    +--- org.checkerframework:checker-compat-qual:2.5.2
|         |    |    |    |    |    |    |    +--- com.google.errorprone:error_prone_annotations:2.2.0
|         |    |    |    |    |    |    |    +--- com.google.j2objc:j2objc-annotations:1.1
|         |    |    |    |    |    |    |    \--- org.codehaus.mojo:animal-sniffer-annotations:1.17
|         |    |    |    |    |    |    \--- javax.validation:validation-api:1.1.0.Final
|         |    |    |    |    |    +--- org.slf4j:slf4j-ext:1.7.28
|         |    |    |    |    |    |    \--- org.slf4j:slf4j-api:1.7.28
|         |    |    |    |    |    +--- org.slf4j:slf4j-api:1.7.28
|         |    |    |    |    |    \--- commons-io:commons-io:2.4
|         |    |    |    |    +--- io.swagger:swagger-compat-spec-parser:1.0.47
|         |    |    |    |    |    +--- io.swagger:swagger-core:1.5.24 (*)
|         |    |    |    |    |    +--- io.swagger:swagger-parser:1.0.47 (*)
|         |    |    |    |    |    +--- com.github.java-json-tools:json-schema-validator:2.2.8
|         |    |    |    |    |    |    +--- com.github.java-json-tools:json-schema-core:1.2.8
|         |    |    |    |    |    |    |    +--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.2
|         |    |    |    |    |    |    |    +--- org.mozilla:rhino:1.7R4
|         |    |    |    |    |    |    |    +--- com.github.fge:jackson-coreutils:1.8
|         |    |    |    |    |    |    |    |    +--- com.github.fge:msg-simple:1.1
|         |    |    |    |    |    |    |    |    |    +--- com.github.fge:btf:1.2
|         |    |    |    |    |    |    |    |    |    |    \--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.2
|         |    |    |    |    |    |    |    |    |    \--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.2
|         |    |    |    |    |    |    |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.2.3 -> 2.10.2 (*)
|         |    |    |    |    |    |    |    |    +--- com.google.guava:guava:16.0.1 -> 27.0.1-android (*)
|         |    |    |    |    |    |    |    |    \--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.2
|         |    |    |    |    |    |    |    \--- com.github.fge:uri-template:0.9
|         |    |    |    |    |    |    |         +--- com.github.fge:msg-simple:1.1 (*)
|         |    |    |    |    |    |    |         +--- com.google.guava:guava:16.0.1 -> 27.0.1-android (*)
|         |    |    |    |    |    |    |         \--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.2
|         |    |    |    |    |    |    +--- javax.mail:mailapi:1.4.3
|         |    |    |    |    |    |    |    \--- javax.activation:activation:1.1
|         |    |    |    |    |    |    +--- joda-time:joda-time:2.9.7
|         |    |    |    |    |    |    +--- com.googlecode.libphonenumber:libphonenumber:8.0.0
|         |    |    |    |    |    |    +--- com.google.code.findbugs:jsr305:3.0.1 -> 3.0.2
|         |    |    |    |    |    |    \--- net.sf.jopt-simple:jopt-simple:5.0.3
|         |    |    |    |    |    +--- com.github.fge:json-patch:1.6
|         |    |    |    |    |    |    +--- com.github.fge:jackson-coreutils:1.6 -> 1.8 (*)
|         |    |    |    |    |    |    \--- com.google.code.findbugs:jsr305:2.0.1 -> 3.0.2
|         |    |    |    |    |    \--- org.apache.httpcomponents:httpclient:4.5.2
|         |    |    |    |    |         +--- org.apache.httpcomponents:httpcore:4.4.4
|         |    |    |    |    |         +--- commons-logging:commons-logging:1.2
|         |    |    |    |    |         \--- commons-codec:commons-codec:1.9
|         |    |    |    |    +--- io.swagger.core.v3:swagger-models:2.0.10 (*)
|         |    |    |    |    +--- io.swagger.parser.v3:swagger-parser-core:2.0.15
|         |    |    |    |    |    \--- io.swagger.core.v3:swagger-models:2.0.10 (*)
|         |    |    |    |    \--- io.swagger.parser.v3:swagger-parser-v3:2.0.15
|         |    |    |    |         +--- io.swagger.core.v3:swagger-models:2.0.10 (*)
|         |    |    |    |         +--- io.swagger.core.v3:swagger-core:2.0.10 (*)
|         |    |    |    |         +--- io.swagger.parser.v3:swagger-parser-core:2.0.15 (*)
|         |    |    |    |         +--- commons-io:commons-io:2.4
|         |    |    |    |         +--- com.fasterxml.jackson.core:jackson-annotations:2.9.10 -> 2.10.2
|         |    |    |    |         +--- com.fasterxml.jackson.core:jackson-databind:2.9.10 -> 2.10.2 (*)
|         |    |    |    |         \--- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.10 -> 2.10.2 (*)
|         |    |    |    +--- io.swagger.parser.v3:swagger-parser-v3:2.0.15 (*)
|         |    |    |    +--- org.slf4j:slf4j-ext:1.7.28 (*)
|         |    |    |    +--- org.slf4j:slf4j-api:1.7.28
|         |    |    |    \--- commons-io:commons-io:2.4
|         |    |    +--- com.samskivert:jmustache:1.14
|         |    |    +--- com.github.jknack:handlebars:4.1.2
|         |    |    |    \--- org.slf4j:slf4j-api:1.7.25 -> 1.7.28
|         |    |    +--- com.github.jknack:handlebars-jackson2:4.1.2
|         |    |    |    +--- com.github.jknack:handlebars:4.1.2 (*)
|         |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.9.7 -> 2.10.2 (*)
|         |    |    +--- commons-io:commons-io:2.4
|         |    |    +--- org.slf4j:slf4j-ext:1.7.12 -> 1.7.28 (*)
|         |    |    +--- org.slf4j:slf4j-api:1.7.12 -> 1.7.28
|         |    |    +--- org.apache.commons:commons-lang3:3.4 -> 3.7
|         |    |    +--- commons-cli:commons-cli:1.2
|         |    |    +--- com.google.guava:guava:26.0-jre -> 27.0.1-android (*)
|         |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-guava:2.9.10 -> 2.10.2
|         |    |    |    +--- com.google.guava:guava:20.0 -> 27.0.1-android (*)
|         |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.10.2
|         |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.10.2 (*)
|         |    |    +--- com.atlassian.commonmark:commonmark:0.11.0
|         |    |    +--- com.github.mifmif:generex:1.0.2
|         |    |    |    \--- dk.brics.automaton:automaton:1.11-8
|         |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.9.10 (*)
|         |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-joda:2.9.10
|         |    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.10 -> 2.10.2
|         |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.10 -> 2.10.2
|         |    |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.9.10 -> 2.10.2 (*)
|         |    |    |    \--- joda-time:joda-time:2.7 -> 2.9.7
|         |    |    +--- com.github.joschi.jackson:jackson-datatype-threetenbp:2.9.10
|         |    |    |    +--- org.threeten:threetenbp:1.4.0
|         |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.10 -> 2.10.2
|         |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.9.10 -> 2.10.2 (*)
|         |    |    \--- org.openapitools:openapi-generator-core:4.2.1
|         |    |         +--- com.google.guava:guava:26.0-jre -> 27.0.1-android (*)
|         |    |         \--- org.slf4j:slf4j-api:1.7.12 -> 1.7.28
|         |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.20
|         |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.20
|         |         |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.20
|         |         |    \--- org.jetbrains:annotations:13.0
|         |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.20
|         |              \--- org.jetbrains.kotlin:kotlin-stdlib:1.3.20 (*)
|         +--- org.yaml:snakeyaml:1.24
|         +--- com.github.spullara.mustache.java:compiler:0.9.0
|         \--- commons-io:commons-io:2.0.1 -> 2.4
\--- com.palantir.revapi:com.palantir.revapi.gradle.plugin:1.1.3
     \--- com.palantir.gradle.revapi:gradle-revapi:1.1.3
          +--- com.google.guava:guava:23.1-jre -> 27.0.1-android (*)
          +--- org.revapi:revapi-basic-features:0.8.1
          |    +--- org.slf4j:slf4j-api:1.7.5 -> 1.7.28
          |    \--- org.revapi:revapi:0.11.1
          |         +--- org.slf4j:slf4j-api:1.7.5 -> 1.7.28
          |         \--- org.jboss:jboss-dmr:1.2.0.Final
          +--- org.revapi:revapi-java:0.19.1
          |    +--- org.revapi:revapi:0.11.1 (*)
          |    \--- org.revapi:revapi-java-spi:0.18.1
          |         \--- org.revapi:revapi:0.11.1 (*)
          +--- org.revapi:revapi-reporter-text:0.10.1
          |    +--- org.revapi:revapi:0.11.1 (*)
          |    \--- org.freemarker:freemarker:2.3.23 -> 2.3.29
          +--- org.freemarker:freemarker:2.3.29
          +--- com.fasterxml.jackson.core:jackson-databind:2.10.2 (*)
          +--- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.2 (*)
          +--- com.fasterxml.jackson.datatype:jackson-datatype-guava:2.10.2 (*)
          \--- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.10.2
               +--- com.fasterxml.jackson.core:jackson-core:2.10.2
               \--- com.fasterxml.jackson.core:jackson-databind:2.10.2 (*)
```

**Comment by ghost** [comment]
I was looking into adding this to [Apache Geode](github.com/apache/geode)'s build and ran into the same issue. 
```
./gradlew --version

------------------------------------------------------------
Gradle 5.4
------------------------------------------------------------

Build time:   2019-04-16 02:44:16 UTC
Revision:     a4f3f91a30d4e36d82cc7592c4a0726df52aba0d

Kotlin:       1.3.21
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          1.8.0_172 (Oracle Corporation 25.172-b11)
OS:           Mac OS X 10.15.2 x86_64
```
```
./gradlew buildEnvironment

> Task :buildEnvironment

------------------------------------------------------------
Root project
------------------------------------------------------------

classpath
+--- com.palantir.revapi:com.palantir.revapi.gradle.plugin:1.2.0
|    \--- com.palantir.gradle.revapi:gradle-revapi:1.2.0
|         +--- com.google.guava:guava:23.1-jre
|         |    +--- com.google.code.findbugs:jsr305:1.3.9 -> 3.0.0
|         |    +--- com.google.errorprone:error_prone_annotations:2.0.18
|         |    +--- com.google.j2objc:j2objc-annotations:1.1
|         |    \--- org.codehaus.mojo:animal-sniffer-annotations:1.14
|         +--- org.revapi:revapi-basic-features:0.8.1
|         |    +--- org.slf4j:slf4j-api:1.7.5 -> 1.7.12
|         |    \--- org.revapi:revapi:0.11.1
|         |         +--- org.slf4j:slf4j-api:1.7.5 -> 1.7.12
|         |         \--- org.jboss:jboss-dmr:1.2.0.Final
|         +--- org.revapi:revapi-java:0.19.1
|         |    +--- org.revapi:revapi:0.11.1 (*)
|         |    \--- org.revapi:revapi-java-spi:0.18.1
|         |         \--- org.revapi:revapi:0.11.1 (*)
|         +--- org.revapi:revapi-reporter-text:0.10.1
|         |    +--- org.revapi:revapi:0.11.1 (*)
|         |    \--- org.freemarker:freemarker:2.3.23 -> 2.3.29
|         +--- org.freemarker:freemarker:2.3.29
|         +--- com.fasterxml.jackson.core:jackson-databind:2.10.2
|         |    +--- com.fasterxml.jackson.core:jackson-annotations:2.10.2
|         |    \--- com.fasterxml.jackson.core:jackson-core:2.10.2
|         +--- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.2
|         |    +--- org.yaml:snakeyaml:1.24
|         |    \--- com.fasterxml.jackson.core:jackson-core:2.10.2
|         +--- com.fasterxml.jackson.datatype:jackson-datatype-guava:2.10.2
|         |    +--- com.google.guava:guava:20.0 -> 23.1-jre (*)
|         |    +--- com.fasterxml.jackson.core:jackson-core:2.10.2
|         |    \--- com.fasterxml.jackson.core:jackson-databind:2.10.2 (*)
|         \--- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.10.2
|              +--- com.fasterxml.jackson.core:jackson-core:2.10.2
|              \--- com.fasterxml.jackson.core:jackson-databind:2.10.2 (*)
\--- nebula.facet:nebula.facet.gradle.plugin:7.0.7
     \--- com.netflix.nebula:nebula-project-plugin:7.0.7
          +--- com.netflix.nebula:nebula-gradle-interop:1.0.2
          |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.70
          |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.2.70
          |              +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.2.70
          |              \--- org.jetbrains:annotations:13.0
          +--- com.netflix.nebula:nebula-publishing-plugin:9.0.2
          |    +--- com.netflix.nebula:gradle-info-plugin:4.0.2
          |    |    +--- com.perforce:p4java:2015.2.1365273
          |    |    |    \--- com.jcraft:jzlib:1.1.2
          |    |    +--- com.netflix.nebula:gradle-contacts-plugin:4.0.2
          |    |    +--- org.eclipse.jgit:org.eclipse.jgit:5.0.1.201806211838-r
          |    |    |    +--- com.jcraft:jsch:0.1.54
          |    |    |    +--- com.jcraft:jzlib:1.1.1 -> 1.1.2
          |    |    |    +--- com.googlecode.javaewah:JavaEWAH:1.1.6
          |    |    |    +--- org.apache.httpcomponents:httpclient:4.5.2
          |    |    |    |    +--- org.apache.httpcomponents:httpcore:4.4.4
          |    |    |    |    +--- commons-logging:commons-logging:1.2
          |    |    |    |    \--- commons-codec:commons-codec:1.9
          |    |    |    \--- org.slf4j:slf4j-api:1.7.2 -> 1.7.12
          |    |    \--- org.tmatesoft.svnkit:svnkit:1.8.12
          |    |         +--- com.jcraft:jsch.agentproxy.svnkit-trilead-ssh2:0.0.7
          |    |         |    +--- com.trilead:trilead-ssh2:1.0.0-build217 -> 1.0.0-build220
          |    |         |    \--- com.jcraft:jsch.agentproxy.core:0.0.7
          |    |         +--- com.trilead:trilead-ssh2:1.0.0-build220
          |    |         +--- net.java.dev.jna:jna-platform:4.1.0
          |    |         |    \--- net.java.dev.jna:jna:4.1.0
          |    |         +--- net.java.dev.jna:jna:4.1.0
          |    |         +--- com.jcraft:jsch.agentproxy.connector-factory:0.0.7
          |    |         |    +--- com.jcraft:jsch.agentproxy.core:0.0.7
          |    |         |    +--- com.jcraft:jsch.agentproxy.usocket-jna:0.0.7
          |    |         |    |    +--- com.jcraft:jsch.agentproxy.core:0.0.7
          |    |         |    |    +--- net.java.dev.jna:jna:3.4.0 -> 4.1.0
          |    |         |    |    \--- net.java.dev.jna:platform:3.4.0
          |    |         |    +--- com.jcraft:jsch.agentproxy.usocket-nc:0.0.7
          |    |         |    |    \--- com.jcraft:jsch.agentproxy.core:0.0.7
          |    |         |    +--- com.jcraft:jsch.agentproxy.sshagent:0.0.7
          |    |         |    |    \--- com.jcraft:jsch.agentproxy.core:0.0.7
          |    |         |    \--- com.jcraft:jsch.agentproxy.pageant:0.0.7
          |    |         |         +--- com.jcraft:jsch.agentproxy.core:0.0.7
          |    |         |         +--- net.java.dev.jna:jna:3.4.0 -> 4.1.0
          |    |         |         \--- net.java.dev.jna:platform:3.4.0
          |    |         +--- de.regnis.q.sequence:sequence-library:1.0.3
          |    |         \--- org.tmatesoft.sqljet:sqljet:1.1.10
          |    |              \--- org.antlr:antlr-runtime:3.4
          |    \--- com.netflix.nebula:gradle-contacts-plugin:4.0.2
          +--- com.netflix.nebula:gradle-contacts-plugin:4.0.2
          +--- com.netflix.nebula:gradle-dependency-lock-plugin:6.1.2
          |    +--- com.netflix.nebula:nebula-gradle-interop:1.0.1 -> 1.0.2 (*)
          |    +--- com.netflix.nebula:gradle-scm-plugin:4.1.0
          |    +--- com.netflix.nebula:gradle-metrics-plugin:5.3.0
          |    |    +--- org.projectlombok:lombok:1.16.6
          |    |    +--- com.google.code.findbugs:jsr305:3.0.0
          |    |    +--- com.google.guava:guava:18.0 -> 23.1-jre (*)
          |    |    +--- joda-time:joda-time:2.8.2
          |    |    +--- com.fasterxml.jackson.core:jackson-core:2.3.2 -> 2.10.2
          |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.3.2 -> 2.10.2 (*)
          |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.3.2 -> 2.10.2
          |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-joda:2.3.2
          |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.3.2 -> 2.10.2
          |    |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.3.2 -> 2.10.2 (*)
          |    |    |    \--- joda-time:joda-time:2.1 -> 2.8.2
          |    |    +--- net.logstash.logback:logstash-logback-encoder:4.5.1
          |    |    |    +--- ch.qos.logback:logback-core:1.1.3
          |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.5.4 -> 2.10.2 (*)
          |    |    +--- io.searchbox:jest:0.1.7
          |    |    |    +--- io.searchbox:jest-common:0.1.7
          |    |    |    |    +--- com.google.guava:guava:18.0 -> 23.1-jre (*)
          |    |    |    |    +--- com.google.code.gson:gson:2.3.1
          |    |    |    |    \--- org.apache.commons:commons-lang3:3.4
          |    |    |    +--- org.apache.httpcomponents:httpcore-nio:4.4.1
          |    |    |    |    \--- org.apache.httpcomponents:httpcore:4.4.1 -> 4.4.4
          |    |    |    +--- org.apache.httpcomponents:httpclient:4.5 -> 4.5.2 (*)
          |    |    |    +--- org.apache.httpcomponents:httpasyncclient:4.1
          |    |    |    \--- org.slf4j:slf4j-api:1.7.12
          |    |    +--- org.elasticsearch:elasticsearch:1.7.2
          |    |    |    +--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    +--- org.apache.lucene:lucene-analyzers-common:4.10.4
          |    |    |    |    \--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    +--- org.apache.lucene:lucene-queries:4.10.4
          |    |    |    |    \--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    +--- org.apache.lucene:lucene-memory:4.10.4
          |    |    |    |    \--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    +--- org.apache.lucene:lucene-highlighter:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-memory:4.10.4 (*)
          |    |    |    |    \--- org.apache.lucene:lucene-queries:4.10.4 (*)
          |    |    |    +--- org.apache.lucene:lucene-queryparser:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-queries:4.10.4 (*)
          |    |    |    |    \--- org.apache.lucene:lucene-sandbox:4.10.4
          |    |    |    |         \--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    +--- org.apache.lucene:lucene-sandbox:4.10.4 (*)
          |    |    |    +--- org.apache.lucene:lucene-suggest:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-analyzers-common:4.10.4 (*)
          |    |    |    |    +--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-misc:4.10.4
          |    |    |    |    |    \--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    |    \--- org.apache.lucene:lucene-queries:4.10.4 (*)
          |    |    |    +--- org.apache.lucene:lucene-misc:4.10.4 (*)
          |    |    |    +--- org.apache.lucene:lucene-join:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    |    \--- org.apache.lucene:lucene-grouping:4.10.4
          |    |    |    |         +--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    |         \--- org.apache.lucene:lucene-queries:4.10.4 (*)
          |    |    |    +--- org.apache.lucene:lucene-grouping:4.10.4 (*)
          |    |    |    +--- org.apache.lucene:lucene-spatial:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-core:4.10.4
          |    |    |    |    +--- org.apache.lucene:lucene-queries:4.10.4 (*)
          |    |    |    |    \--- com.spatial4j:spatial4j:0.4.1
          |    |    |    \--- org.yaml:snakeyaml:1.12 -> 1.24
          |    |    +--- org.apache.httpcomponents:fluent-hc:4.5.1
          |    |    |    +--- org.apache.httpcomponents:httpclient:4.5.1 -> 4.5.2 (*)
          |    |    |    \--- commons-logging:commons-logging:1.2
          |    |    +--- org.apache.commons:commons-lang3:3.4
          |    |    +--- commons-io:commons-io:2.5
          |    |    \--- com.netflix.nebula:gradle-info-plugin:3.+ -> 4.0.2 (*)
          |    +--- org.codehaus.groovy.modules.http-builder:http-builder:0.7.1
          |    |    +--- org.apache.httpcomponents:httpclient:4.2.1 -> 4.5.2 (*)
          |    |    +--- net.sf.json-lib:json-lib:2.3
          |    |    |    +--- commons-beanutils:commons-beanutils:1.8.0
          |    |    |    |    \--- commons-logging:commons-logging:1.1.1 -> 1.2
          |    |    |    +--- commons-collections:commons-collections:3.2.1
          |    |    |    +--- commons-lang:commons-lang:2.4
          |    |    |    +--- commons-logging:commons-logging:1.1.1 -> 1.2
          |    |    |    \--- net.sf.ezmorph:ezmorph:1.0.6
          |    |    |         \--- commons-lang:commons-lang:2.3 -> 2.4
          |    |    +--- net.sourceforge.nekohtml:nekohtml:1.9.16
          |    |    |    \--- xerces:xercesImpl:2.9.1
          |    |    \--- xml-resolver:xml-resolver:1.2
          |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.70 (*)
          \--- com.netflix.nebula:gradle-info-plugin:4.0.2 (*)

(*) - dependencies omitted (listed previously)
```

**Comment by randomanderson** [comment]
Ran into the exact same issue.  Tried overriding the Guava dependency in multiple different ways but was unsuccessful.

**Comment by iamdanfox** [comment]
@randomanderson we've [eliminated](https://github.com/palantir/gradle-revapi/pull/214) the guava dependency. Wanna try gradle-revapi 1.3.2?

**Comment by randomanderson** [comment]
@iamdanfox Thanks for the quick turnaround!  My gradle build is unable to resolve 1.3.2.  When I check on the gradle plugin site (https://plugins.gradle.org/plugin/com.palantir.revapi) the highest version is 1.3.0.

Is there another repository that I should be adding?

**Comment by iamdanfox** [comment]
Ah yes there was a CVE reported with gradle plugin publishing, so we upgraded and I've just triggered 1.4.0. As soon as this builds finishes, you should be good to go: https://circleci.com/workflow-run/d82fa9c4-c9b6-464d-b100-9a2b3cb5aba5 

aaand it's up. https://plugins.gradle.org/plugin/com.palantir.revapi

**Comment by randomanderson** [comment]
Works perfectly.  Thanks!

**Comment by randomanderson** [comment]
Just wanted to say thanks again.  This plugin helped me validate [this massive PR](https://github.com/DataDog/dd-trace-java/pull/1354)

**Comment by CRogers** [comment]
Fixed by #214

**Comment by iamdanfox** [comment]
Oh nice! we've also found revapi pretty helpful for keeping us honest while iterating on our own tracing library (https://github.com/palantir/tracing-java), especially because it's in use _everywhere_ so ABI/API breaks are pretty disastrous!
